applications/worker/templates/deployment.yaml

```diff
@@ -1,10 +1,5 @@
-{{- if and (.Values.deploymentStrategy) (eq .Values.deploymentStrategy.kind "blueGreen") }}
```

Comment:
Is this still needed in the web template?

Reply:
yes to web template, no to worker